### ISSUE TITLE
You have requested a non-existent service "liip_functional_test.validator".

### ISSUE THREAD

**Issue by Chrysweel** [liip/LiipFunctionalTestBundle#270]
Hello.

I just installed this bundle, with instructions of the documentation.

And with test of example I get the following error:

```
Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException: You have requested a non-existent service "liip_functional_test.validator".

/mypath/app/bootstrap.php.cache:2063
/mypath/vendor/liip/functional-test-bundle/Utils/HttpAssertions.php:90
/mypath/vendor/liip/functional-test-bundle/Test/WebTestCase.php:882
```

In my composer.json I have:

```
"symfony/symfony" : "2.3.*",
"liip/functional-test-bundle" : "^1.6"
```

AppKernel.

```
        if (in_array($this->getEnvironment(), array('dev', 'test'))) {
            // $bundles[] = new Symfony\Bundle\WebProfilerBundle\WebProfilerBundle();
            $bundles[] = new Sensio\Bundle\DistributionBundle\SensioDistributionBundle();
            $bundles[] = new Sensio\Bundle\GeneratorBundle\SensioGeneratorBundle();
            $bundles[] = new Liip\FunctionalTestBundle\LiipFunctionalTestBundle();
        }
```

Thanks in advance !

**Comment by alexislefebvre** [comment]
Can you please add the code from your test?

**Comment by Chrysweel** [comment]
Yes.

```
<?php

namespace MyProject\CoreBundle\Tests\Controller;

use Liip\FunctionalTestBundle\Test\WebTestCase;

class LipControllerTest extends WebTestCase
{
    public function testContact()
    {
        $client = static::makeClient();
        $crawler = $client->request('GET', '/contacto');
        $this->assertStatusCode(200, $client);
        $form = $crawler->selectButton('Enviar')->form();
        $crawler = $client->submit($form);

        // We should get a validation error for the empty fields.
        $this->assertStatusCode(200, $client);
        $this->assertValidationErrors(['data.email', 'data.message'], $client->getContainer());

        // Try again with with the fields filled out.
        $form = $crawler->selectButton('Submit')->form();
        $form->setValues([
        		'contact[name]'  	=> 'Hello',
        		'contact[email]' 	=> 'nobody@example.com',
        		'contact[subject]'	=> 'asunto',
        		'contact[body]'		=> 'body'
        		
        ]);
        $client->submit($form);
        $this->assertStatusCode(302, $client);
    }
}
```

**Comment by alexislefebvre** [comment]
I wrote a wrong version previously, sorry. Can you please try with `"liip/functional-test-bundle" : "1.6.3"`?

**Comment by alexislefebvre** [comment]
I just saw this note in the README.md:

> Note: Both `assertStatusCode` and `assertValidationErrors` only works on Symfony 2.5+

I don't have any solution, sorry.

**Comment by Chrysweel** [comment]
Yes I tried with version 1.6.3.

If this version need symfony 2.5 composer of this bundle would require version symfony 2.5...

And the version to symfony 2.3 dont have this method.. to avoid this errors.

Thanks for help @alexislefebvre  !

**Comment by alexislefebvre** [comment]
I reopen this since the Bundle should display a warning instead of displaying an error.

**Comment by alexislefebvre** [comment]
Fixed by #367.
